### DISCUSSION TITLE
Rules to prefer for_each instead of count?

### DISCUSSION THREAD

**Discussion by mloskot** [terraform-linters/tflint#1990] [Q&A]
Is there any way to specify TFLint to warn about uses of `count` unless specifically annotated to be ignored?

That would be very helpful for a project with convention to always prefer `for_each` and resort to `count` in few special cases.

**Comment by wata727** [comment]
A good place to start is with the OPA ruleset.
https://github.com/terraform-linters/tflint-ruleset-opa
